packages/fx-core/src/plugins/resource/frontend/dotnet/constants.ts

```diff
@@ -54,15 +52,19 @@ export class DotnetCommands {
     `dotnet publish --configuration Release --runtime ${runtime} --self-contained`;
 }
 
+export class ScaffoldInfo {
```

Comment:
Since there is already an interface of CodeTemplateInfo, why don't we leverage this?